.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,58 @@
+name: nodejs
+
+on:
+  push:
+    paths:
+      - "samples/nodejs/*"
+      - ".github/workflows/nodejs.yml"
```

Comment:
Can you add this for the other sample too, so the workflow is triggered when it's changed.